### PR TITLE
Infinex Patron NFT deployment toml

### PR DESCRIPTION
**Pull request by rayInfinex**

### CODE REVIEW

File: infinex-patron/nft/infinex-patron-nft.toml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+target = "infinex:1.0.0@patron-nft"
+tags = ["1.0.0"]
+options.PATRON_NFT_DEPLOYER = ""
+options.PATRON_NFT_OWNER = ""
+options.PATRON_NFT_SALT = "PATRON-NFT"
+options.PATRON_NFT_NAME = "Infinex Patrons"
+options.PATRON_NFT_SYMBOL = "XPATRON"
+options.PATRON_NFT_MINT_DESTINATION = "<%= imports.InfinexPatronDistributor.contracts.PatronDistributor.address %>"
+options.PATRON_NFT_ROYALTY_RECEIVER = ""
+options.PATRON_NFT_FEE_NUMERATOR = "0"
```
Comment (rayInfinex) [inline infinex-patron/nft/infinex-patron-nft.toml:21]: this should be 500 I believe (5% royalty)

File: infinex-patron/nft/infinex-patron-nft.toml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+source = "infinex:1.0.0@patron-nft"
+target = "infinex:1.0.0@patron-nft"
+tags = ["1.0.0"]
+options.PATRON_NFT_DEPLOYER = ""
+options.PATRON_NFT_OWNER = ""
+options.PATRON_NFT_SALT = "PATRON-NFT"
+options.PATRON_NFT_NAME = "Infinex Patrons"
+options.PATRON_NFT_SYMBOL = "XPATRON"
+options.PATRON_NFT_MINT_DESTINATION = "<%= imports.InfinexPatronDistributor.contracts.PatronDistributor.address %>"
+options.PATRON_NFT_ROYALTY_RECEIVER = ""
```
Comment (rayInfinex) [inline infinex-patron/nft/infinex-patron-nft.toml:20]: this needs to be set

**Comment by zeus-infinex** [comment]
## [testnets](https://sys.infinex.xyz/reports/evm-contracts/deployments-test/de03f7e/report/testnets/) <span title="passed">✅214</span> | <span title="failed">❌12</span> | <span title="skipped">⏩6</span>
[Markdown](https://sys.infinex.xyz/reports/evm-contracts/deployments-test/de03f7e/report/testnets/report.md) | [JSON](https://sys.infinex.xyz/reports/evm-contracts/deployments-test/de03f7e/report/testnets/report.json)
<details>
<summary>gp</summary>

### GovernancePoints (base testnets) GovernancePoints has authorised transfers only enabled
https://github.com/infinex-xyz/infinex-deployments/blob/main/workspace/test/tests/gp.test.ts#L84

```
AssertionError: expected false to be true // Object.is equality
```
</details>
<details>
<summary>multichain</summary>

### Multichain (blast testnets) Initial Deployment Infinex Protocol Config Beacon points to valid USDC contract
https://github.com/infinex-xyz/infinex-deployments/blob/main/workspace/test/tests/multichain.test.ts#L183

```
AssertionError: expected '0x42000000000000000000000000000000000…' to be '0x43000000000000000000000000000000000…' // Object.is equality
```
</details>
<details>
<summary>patron-app</summary>

### Patron Purchase (arbitrum testnets) Check Ownership Transferred Successfully App Beacon
https://github.com/infinex-xyz/infinex-deployments/blob/main/workspace/test/tests/patron-app.test.ts#L120

```
AssertionError: expected '0x6298551D56F825B3b6c3350D32129Ffd3cF…' to be undefined // Object.is equality
```

### Patron Purchase (arbitrum testnets) Check Ownership Transferred Successfully Patron Point of Purchase
https://github.com/infinex-xyz/infinex-deployments/blob/main/workspace/test/tests/patron-app.test.ts#L126

```
AssertionError: expected '0x6298551D56F825B3b6c3350D32129Ffd3cF…' to be undefined // Object.is equality
```

### Patron Purchase (base testnets) Check Ownership Transferred Successfully App Beacon
https://github.com/infinex-xyz/infinex-deployments/blob/main/workspace/test/tests/patron-app.test.ts#L120

```
AssertionError: expected '0x6298551D56F825B3b6c3350D32129Ffd3cF…' to be undefined // Object.is equality
```

### Patron Purchase (base testnets) Check Ownership Transferred Successfully Patron Point of Purchase
https://github.com/infinex-xyz/infinex-deployments/blob/main/workspace/test/tests/patron-app.test.ts#L126

```
AssertionError: expected '0x6298551D56F825B3b6c3350D32129Ffd3cF…' to be undefined // Object.is equality
```

### Patron Purchase (ethereum testnets) Check Ownership Transferred Successfully App Beacon
https://github.com/infinex-xyz/infinex-deployments/blob/main/workspace/test/tests/patron-app.test.ts#L120

```
AssertionError: expected '0x6298551D56F825B3b6c3350D32129Ffd3cF…' to be undefined // Object.is equality
```

### Patron Purchase (ethereum testnets) Check Ownership Transferred Successfully Patron Point of Purchase
https://github.com/infinex-xyz/infinex-deployments/blob/main/workspace/test/tests/patron-app.test.ts#L126

```
AssertionError: expected '0x6298551D56F825B3b6c3350D32129Ffd3cF…' to be undefined // Object.is equality
```

### Patron Purchase (optimism testnets) Check Ownership Transferred Successfully App Beacon
https://github.com/infinex-xyz/infinex-deployments/blob/main/workspace/test/tests/patron-app.test.ts#L120

```
AssertionError: expected '0x6298551D56F825B3b6c3350D32129Ffd3cF…' to be undefined // Object.is equality
```

### Patron Purchase (optimism testnets) Check Ownership Transferred Successfully Patron Point of Purchase
https://github.com/infinex-xyz/infinex-deployments/blob/main/workspace/test/tests/patron-app.test.ts#L126

```
AssertionError: expected '0x6298551D56F825B3b6c3350D32129Ffd3cF…' to be undefined // Object.is equality
```

### Patron Purchase (polygon testnets) Check Ownership Transferred Successfully App Beacon
https://github.com/infinex-xyz/infinex-deployments/blob/main/workspace/test/tests/patron-app.test.ts#L120

```
AssertionError: expected '0x6298551D56F825B3b6c3350D32129Ffd3cF…' to be undefined // Object.is equality
```

### Patron Purchase (polygon testnets) Check Ownership Transferred Successfully Patron Point of Purchase
https://github.com/infinex-xyz/infinex-deployments/blob/main/workspace/test/tests/patron-app.test.ts#L126

```
AssertionError: expected '0x6298551D56F825B3b6c3350D32129Ffd3cF…' to be undefined // Object.is equality
```
</details>


## [staging](https://sys.infinex.xyz/reports/evm-contracts/deployments-test/de03f7e/report/staging/) <span title="passed">✅229</span> | <span title="failed">❌2</span> | <span title="skipped">⏩6</span>
[Markdown](https://sys.infinex.xyz/reports/evm-contracts/deployments-test/de03f7e/report/staging/report.md) | [JSON](https://sys.infinex.xyz/reports/evm-contracts/deployments-test/de03f7e/report/staging/report.json)
<details>
<summary>gp</summary>

### GovernancePoints (base staging) GovernancePoints has authorised transfers only enabled
https://github.com/infinex-xyz/infinex-deployments/blob/main/workspace/test/tests/gp.test.ts#L84

```
AssertionError: expected false to be true // Object.is equality
```
</details>
<details>
<summary>multichain</summary>

### Multichain (blast staging) Initial Deployment Infinex Protocol Config Beacon has correct solana configuration
https://github.com/infinex-xyz/infinex-deployments/blob/main/workspace/test/tests/multichain.test.ts#L285

```
ContractFunctionExecutionError: HTTP request failed.
```
</details>


## [mainnets](https://sys.infinex.xyz/reports/evm-contracts/deployments-test/de03f7e/report/mainnets/) <span title="passed">✅225</span> | <span title="failed">❌6</span> | <span title="skipped">⏩6</span>
[Markdown](https://sys.infinex.xyz/reports/evm-contracts/deployments-test/de03f7e/report/mainnets/report.md) | [JSON](https://sys.infinex.xyz/reports/evm-contracts/deployments-test/de03f7e/report/mainnets/report.json)
<details>
<summary>gp</summary>

### GovernancePoints (base mainnets) GovernancePoints has authorised transfers only enabled
https://github.com/infinex-xyz/infinex-deployments/blob/main/workspace/test/tests/gp.test.ts#L84

```
AssertionError: expected false to be true // Object.is equality
```

### GovernancePoints (base mainnets) GovernancePoints has the GPDistributor added as an authorised transferrer
https://github.com/infinex-xyz/infinex-deployments/blob/main/workspace/test/tests/gp.test.ts#L98

```
AssertionError: expected false to be true // Object.is equality
```

### GovernancePoints (base mainnets) GPDistributor has the correct owner
https://github.com/infinex-xyz/infinex-deployments/blob/main/workspace/test/tests/gp.test.ts#L116

```
ContractFunctionExecutionError: The contract function "owner" returned no data ("0x").
```

### GovernancePoints (base mainnets) GPDistributor has the correct GPToken address
https://github.com/infinex-xyz/infinex-deployments/blob/main/workspace/test/tests/gp.test.ts#L122

```
ContractFunctionExecutionError: The contract function "GPToken" returned no data ("0x").
```

### GovernancePoints (base mainnets) GPDistributor has the correct authorised keeper
https://github.com/infinex-xyz/infinex-deployments/blob/main/workspace/test/tests/gp.test.ts#L126

```
ContractFunctionExecutionError: The contract function "authorizedKeepers" returned no data ("0x").
```
</details>
<details>
<summary>multichain</summary>

### Multichain (blast mainnets) Initial Deployment Infinex Protocol Config Beacon is set to itself as latest
https://github.com/infinex-xyz/infinex-deployments/blob/main/workspace/test/tests/multichain.test.ts#L229

```
ContractFunctionExecutionError: HTTP request failed.
```
</details>

File: infinex-patron/nft/infinex-patron-nft.toml
Comment (rayInfinex) [inline infinex-patron/nft/infinex-patron-nft.toml:1]: infinex-patron/nft/infinex-patron-nft.ethereum.toml

File: infinex-patron/nft/infinex-patron-nft.toml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+name = "infinex-patron-nft"
```
Comment (rayInfinex) [inline infinex-patron/nft/infinex-patron-nft.toml:1]:
```suggestion
#  pnpm cannon build infinex-patron/nft/infinex-patron-nft.ethereum.toml --chain-id 1 --dry-run
#  pnpm cannon publish infinex-patron-nft:1.0.0@patron-nft --chain-id 1 --exclude-cloned
#  pnpm cannon verify infinex-patron-nft:1.0.0@patron-nft --chain-id 1 --api-key
name = "infinex-patron-nft"
```

File: infinex-patron/nft/infinex-patron-nft.toml [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+
+[pull.InfinexPatronDistributor]
+source = "infinex:1.0.0@patron-distributor"
+
+[clone.InfinexPatronNFT]
+source = "infinex:1.0.0@patron-nft"
+target = "infinex:1.0.0@patron-nft"
+tags = ["1.0.0"]
+options.PATRON_NFT_DEPLOYER = ""
```
Comment (rayInfinex) [inline infinex-patron/nft/infinex-patron-nft.toml:14]: TBD

File: infinex-patron/nft/infinex-patron-nft.toml [hunk 5]
```diff
@@ -0,0 +1,22 @@
+name = "infinex-patron-nft"
+preset = "patron-nft"
+version = "1.0.0"
+description = "The Infinex Patron NFT"
+
+
+[pull.InfinexPatronDistributor]
+source = "infinex:1.0.0@patron-distributor"
```
Comment (rayInfinex) [inline infinex-patron/nft/infinex-patron-nft.toml:8]:
```suggestion
source = "infinex-patron-distributor:1.0.0@patron-distributor"
```

File: infinex-patron/nft/infinex-patron-nft.toml [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+[clone.InfinexPatronNFT]
+source = "infinex:1.0.0@patron-nft"
+target = "infinex:1.0.0@patron-nft"
+tags = ["1.0.0"]
+options.PATRON_NFT_DEPLOYER = ""
+options.PATRON_NFT_OWNER = ""
+options.PATRON_NFT_SALT = "PATRON-NFT"
+options.PATRON_NFT_NAME = "Infinex Patrons"
+options.PATRON_NFT_SYMBOL = "XPATRON"
+options.PATRON_NFT_MINT_DESTINATION = "<%= imports.InfinexPatronDistributor.contracts.PatronDistributor.address %>"
```
Comment (rayInfinex) [inline infinex-patron/nft/infinex-patron-nft.toml:19]: I believe it should be 
```suggestion
options.PATRON_NFT_MINT_DESTINATION = "<%= imports.InfinexPatronDistributor.imports.InfinexPatronDistrutor.contracts.PatronDistributor.address %>"
```

**Pull request closed without merging**